clar2wasm/tests/wasm-generation/blockinfo.rs

```diff
@@ -38,3 +40,30 @@ proptest! {
         }
     }
 }
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_at_block(
+        value in PropValue::any(),
+        buf in buffer(32)
+    ) {
+        crosscheck(
+            &format!("(at-block {buf} {value})"),
+            Ok(Some(value.into()))
+        )
+    }
+
+    #[test]
+    fn crosscheck_at_block_inner(
+        value in PropValue::any(),
+        buf in buffer(32)
+    ) {
+
+        crosscheck(
+            &format!("(begin (at-block {buf} {value}))"),
+            Ok(Some(value.into()))
+        )
+    }
+}
```

Comment:
What does it change to be inside a function? What difference in the code traversal is provoked when we are inside a function? In this case, shouldn't we test if it works inside a function inside a function?

Reply:
It seems there is no diff, for `at-block`, calling it inside a function. Removing those duplicated tests.